app/views/completed_transaction/show.html.erb

```diff
@@ -25,6 +25,11 @@
           <%= link_to 'Sign up', @publication.promotion['url'],
                 title: "Get a text or email reminder when your MOT is due", rel: "external", class: "button", role: "button" %>
         </p>
+      <% elsif @publication.promotion['category'] == 'electric_vehicle' %>
+        <p>Make your next car electric.</p>
+        <p>
+          <%= link_to 'Find out how much you could save.', @publication.promotion['url'],
+                title: "Learn about electric cars", rel: "external", class: "button", role: "button" %>
```

Comment:
I made the `title` bit of this up. The content we were given said "Make your next car electric. Find out how much you could save." with the second sentence as the link, so 🤷‍♀.

Reply:
Should this be `button`? Or `govuk-link`?